WordPress/build.gradle

```diff
@@ -184,8 +184,8 @@ android {
             applicationIdSuffix ".prealpha"
             dimension "buildType"
 
-            versionName versionProperties.getProperty("alpha.versionName")
-            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
+            versionName project.findProperty("installableBuildVersionName") ?: versionProperties.getProperty("alpha.versionName")
+            versionCode 1 // Fixed versionCode because those builds are not meant to be uploaded to the PlayStore.
```

Comment:
Just thinking about this one – by doing this we lose the ability to install newer installable builds overtop of older ones. Do we care about that? 

I'm not sure how big those numbers can get, but could we work around this by having the `versionCode` be `new Date().to_int()` or similar? Or is that just overthinking it?

Reply:
That's a good question.

Note though that this was already the case (having the versionCode ne the same for multiple installable builds) before that change, because `versionCode` is only bumped during new betas but there are many PRs done (especially on `develop`) between two betas (and between two merges of the `release` branch to `develop` after a beta). So all those PRs' Installable Builds were already using the same `versionCode` (the one from the last beta since PR was opened), and it didn't seem to be a problem so far? 🤔

Reply:
I think what @jkmassel means is that (correct me if I am wrong), since the previous installable build versions had a higher version code, developers won't be able to update them without first deleting the existing build **once**. After that, I agree that the same version code should allow updates, but I haven't tested the behavior.

I don't think it's a big issue since these are just test builds, but I do like the date idea 😅

Reply:
I am good either way. If it turns out to be an issue, it's easy enough to improve since we are all aware of it. So, whatever you want to go with, it's a 👍 from me.